database/cseed.js

```diff
@@ -0,0 +1,208 @@
+/* eslint-disable no-nested-ternary */
+/* eslint-disable no-plusplus */
+/* eslint-disable func-names */
+const Uuid = require('cassandra-driver').types.Uuid;
+const fs = require('fs');
+const faker = require('faker');
+const os = require('os');
+const moment = require('moment');
+
+console.log(Uuid.random());
+console.log(Uuid.random());
+
+
+let reservationID = 0;
+const intervalOptions = [15, 30, 60];
+const minuteOptions = {
+  15: ['15', '30', '45', '00'],
+  30: ['30', '00'],
+  60: ['00'],
+};
+const openOptions = [7, 8, 9, 10];
+const closeOptions = [20, 21, 22, 23];
+const tableOptions = [2, 4, 6, 8, 10, 20];
+const tableNumberOptions = [3, 5, 7, 9, 11];
+
+const generateRandom = function (options) {
+  return Math.floor(Math.random() * options.length);
+};
+
+const generateMinute = function (interval) {
+  const options = minuteOptions[interval];
+  return options[generateRandom(options)];
+};
+
+const createRestaurant = function (restaurantID) {
+  const restaurantName = `${faker.lorem.word()} ${faker.lorem.word()}`;
+  const openTime = openOptions[generateRandom(openOptions)];
+  const closeTime = closeOptions[generateRandom(closeOptions)];
+  const intervalTime = intervalOptions[generateRandom(intervalOptions)];
+
+  const row = `${restaurantID},${restaurantName},${openTime},${closeTime},${intervalTime}\n`;
+  const output = [];
+  output.push(row);
+  return [output.join(os.EOL), openTime, closeTime, intervalTime];
+};
+
+const createSeating = function (restaurant, number) {
+  const output = [];
+  // some restaurants are very small and have limited seating options
+  let largestTable;
+  let smallestTable;
+  if (number % 100 === 0) {
+    for (let i = 0; i < 3; i++) {
+      const assign = faker.random.number({ min: 0, max: 3 });
+      largestTable = largestTable > tableOptions[i] ? largestTable : tableOptions[i];
+      smallestTable = smallestTable > tableOptions[i] ? smallestTable : tableOptions[i];
+      const row = `${restaurant},${tableOptions[i]},${tableNumberOptions[assign]}`;
+      output.push(row);
+    }
+  } else {
+    for (let j = 0; j < tableOptions.length; j++) {
+      const assign = generateRandom(tableNumberOptions);
+      largestTable = largestTable > tableOptions[j] ? largestTable : tableOptions[j];
+      smallestTable = smallestTable < tableOptions[j] ? smallestTable : tableOptions[j];
+      const row = `${restaurant},${tableOptions[j]},${tableNumberOptions[assign]}`;
+      output.push(row);
+    }
+  }
+  return [output.join('\n') + '\n', largestTable, smallestTable];
+};
+
+
+const createReservation = function (restaurantID, openTime, closeTime, intervalTime, largestTable, number) {
+  let output = [];
+  const start = moment('2019-08-01');
```

Comment:
```suggestion
  const start = moment();
```
This can ensure your db seeding is from where the seeding date is.